.circleci/config.yml

```diff
@@ -75,6 +75,44 @@ jobs:
             if [ -n "${CIRCLE_TAG}" ]; then
               ./bin/deploy-dockerhub.sh "$CIRCLE_TAG"
             fi
+  integration_test:
+    machine:
+      image: ubuntu-1604:201903-01
+    working_directory: ~/kinto-dist
+    steps:
+      - checkout
+      - run:
+          name: "Set Python Version"
+          command: |
+            pyenv global 3.7.0
+      - run:
+          name: Install Docker Compose
+          command: |
+            set -x
+            sudo bash -c "curl -L https://github.com/docker/compose/releases/download/1.11.2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose"
+            sudo chmod +x /usr/local/bin/docker-compose
+      - run:
+          name: Set hosts
+          command: |
+            echo 127.0.0.1 localhost | sudo tee -a /etc/hosts
+            cat /etc/hosts
+      - run:
+          name: Build
+          command: docker-compose build
+      - run:
+          name: Setup env and run tests
+          command: |
+            mkdir mail
+            sudo chmod 777 mail
+            docker-compose run web migrate
+            docker-compose up -d web
+            sleep 5
+            pip install pytest-selenium
+            docker run -d --shm-size 2g --net host selenium/standalone-firefox
+            sleep 5
```

Comment:
ditto: try the curl retry if applicable here, because usually sleeps end up failing one day or another